lerna-debug.log

```diff
@@ -0,0 +1,14 @@
+0 silly argv { _: [ 'clean' ], lernaVersion: '4.0.0', '$0': 'lerna' }
```

Comment:
Add this to gitignore. Also it shows that you've added a nonexistant/invalid package.